### PR TITLE
Features/#450 metadata etrago tables

### PR DESCRIPTION
**Pull request by ClaraBuettner**
@AmeliaNadal I added some metadata for the etrago tables on this branch.

### CODE REVIEW

File: src/egon/data/datasets/etrago_setup.py [hunk 1]
```diff
@@ -388,13 +548,17 @@ class EgonPfHvStore(Base):
         source_dict["bgr_inspee"],
         source_dict["bgr_inspeeds"],
         source_dict["bgr_inspeeds_data_bundle"],
-        source_dict["bgr_inspeeds_data_bundle"],
         source_dict["bgr_inspeeds_report"],
+        source_dict["SciGRID_gas"],
     ]
```
Comment (AmeliaNadal) [inline src/egon/data/datasets/etrago_setup.py:553]: Are there no sources for DSM, heat and e-mobility storages that should figure here?

File: src/egon/data/datasets/etrago_setup.py [hunk 2]
```diff
@@ -129,8 +129,35 @@ def __init__(self, dependencies):
 
 
 class EgonPfHvBus(Base):
+
+    source_list = [
+        sources()["egon-data"],
```
Comment (AmeliaNadal) [inline src/egon/data/datasets/etrago_setup.py:134]: Can we really cite this eGon-data as a source in our metadata?

Comment (ClaraBuettner) [reply src/egon/data/datasets/etrago_setup.py:134]: We added it like this to every table, so I think it's fine.

File: src/egon/data/datasets/etrago_setup.py (hunk 1 above)
Comment (ClaraBuettner) [reply src/egon/data/datasets/etrago_setup.py:553]: Yes you're right. At least for DSM there should be a source. @KathiEsterl could you write which one(s) you would cite? I can integrate it to the scripts.

Comment (KathiEsterl) [reply src/egon/data/datasets/etrago_setup.py:553]: Yes, I can: 

We use the concept and parameters by Wilkos work: https://www.sciencedirect.com/science/article/pii/S2666792420300019
Apart from that, there is no work to be cited, I guess.

I am using the following tables to generate my data:
  sources:
    cts_loadcurves:
      schema: 'demand'
      table: 'egon_etrago_electricity_cts'
    ind_osm_loadcurves:
      schema: 'demand'
      table: 'egon_osm_ind_load_curves'
    ind_osm_loadcurves_individual:
      schema: 'demand'
      table: 'egon_osm_ind_load_curves_individual'
    ind_sites_loadcurves:
      schema: 'demand'
      table: 'egon_sites_ind_load_curves'
    ind_sites_loadcurves_individual:
      schema: 'demand'
      table: 'egon_sites_ind_load_curves_individual'
    ind_sites:
      schema: 'demand'
      table: 'egon_industrial_sites'
    ind_sites_schmidt:
      schema: 'demand'
      table: 'egon_schmidt_industrial_sites'
    demandregio_ind_sites:
      schema: 'demand'
      table: 'egon_demandregio_sites_ind_electricity'
      
I am not completely sure in which format you need the sources, just let me know if I should change anything!

Comment (ClaraBuettner) [reply src/egon/data/datasets/etrago_setup.py:553]: Okay, actually I need to add all the sources that have been used as an input to all the tables you listed.
But I guess all these tables don't have meta data by now, right?
It's not that easy for me to find out which sources have been used. So I would not add them now but leave a comment that they will be added later.

**Comment by KathiEsterl** [comment]
Sources used in PV_ground_mounted -> power plants table -> etrago generators:

mastr (existierende Einheiten)
potential areas (RLI)
vg250 (Bundesland-shapes)
nep (Kapazitäten-Zielwerte pro Bundesland)

**Comment by ClaraBuettner** [comment]
I'm done with the meta data for all etrago tables. 
Does someone wants to check it before I merge it into the other 450-branch? 
I did not checked every metadata json with the OMI-tool, but at least some of them.

**Comment by ClaraBuettner** [comment]
> Sources used in PV_ground_mounted -> power plants table -> etrago generators:
> 
> mastr (existierende Einheiten) potential areas (RLI) vg250 (Bundesland-shapes) nep (Kapazitäten-Zielwerte pro Bundesland)

I added all sources except the potential areas since I didn't know how to cite them.
Is it fine to add them later, as soon as these tables have meta data?

File: src/egon/data/datasets/etrago_setup.py [hunk 3]
```diff
@@ -431,9 +786,38 @@ class EgonPfHvStore(Base):
...
-    __tablename__ = "egon_etrago_store_timeseries"
-    __table_args__ = {"schema": "grid"}
+    source_dict = sources()
+    # TODO: Add other sources for dsm
+    source_list = [
+        source_dict["bgr_inspee"],
+        source_dict["bgr_inspeeds"],
+        source_dict["bgr_inspeeds_data_bundle"],
+        source_dict["bgr_inspeeds_report"],
+        source_dict["SciGRID_gas"],
```
Comment (AmeliaNadal) [inline src/egon/data/datasets/etrago_setup.py:796]: I think these sources are not necessary there.

**Review by AmeliaNadal — APPROVED** [review]

File: src/egon/data/datasets/etrago_setup.py (hunk 3 above)
Comment (ClaraBuettner) [reply src/egon/data/datasets/etrago_setup.py]: I deleted them

**Pull request merged**